example/oracle_connection.tf

```diff
@@ -0,0 +1,26 @@
+provider "equinix" {
+  client_id     = "your_client_id"
+  client_secret = "your_client_secret"
+}
+
+data "equinix_ecx_port" "dot1q-1-pri" {
+  name = "sit-001-CX-DC5-NL-Dot1q-BO-10G-PRI-JUN-27"
+}
+
+data "equinix_ecx_l2_sellerprofile" "oracle" {
+  name = "Oracle Cloud Infrastructure -OCI- FastConnect"
+}
+
+resource "equinix_ecx_l2_connection" "oracle-dot1q" {
+  name                  = "tf-oracle-dot1q"
+  profile_uuid          = var.sp_oracle_uuid
+  speed                 = 50
+  speed_unit            = "MB"
+  notifications         = ["marry@equinix.com", "john@equinix.com"]
+  purchase_order_number = "1234567890"
+  port_uuid             = var.port_dot1q_pri_uuid
+  vlan_stag             = 1700
+  seller_region         = "us-ashburn-1"
+  seller_metro_code     = "DC"
+  authorization_key     = "123456789"
```

Comment:
It may be helpful to specify that for an OCI FC connection the value required in authorization_key is the OCID of an existing Virtual Circuit

`authorization_key     = "ocid1.virtualcircuit.oc1.eu-frankfurt-1.123456789" # Your Virtual Circuit OCID`

Reply:
Agree, I've used example that you have provided with oracle cloud resources. So I was able to referenced id from oci_virtual_circuit